docs/src/assets/scss/carbon-ads.scss

```diff
@@ -4,6 +4,10 @@
      }
 }
 
+.docs-ad {
+    height: 335px;
+}
```

Comment:
I understand why we want to set `height` here, but how did we calculate that it should be `335px`?

Reply:
On the eslint.org homepage, it's set to [290px](https://github.com/eslint/eslint.org/blob/1f3d6e58af1226b3aa62d10cdf9667a608d18c92/src/assets/scss/components/carbon-ads.scss#L13)?

Reply:
@nzakas do we know (or have control over it) what the maximum ad height is?

Reply:
I’m pretty sure the max height is 290px. I remember digging into this when we were in the middle of the redesign. We definitely don’t want different settings between the docs site and the main site. I’ll take a closer look at this next week.